src/test/java/app/bpartners/geojobs/repository/model/community/CommunityDetectableObjectTypeTest.java

```diff
@@ -10,12 +10,12 @@ class CommunityDetectableObjectTypeTest {
   void getter_setter_test() {
     CommunityDetectableObjectType value = new CommunityDetectableObjectType();
     value.setId("123");
-    value.setType(DetectableType.POOL);
+    value.setType(DetectableType.PISCINE);
```

Comment:
When possible, import directly the value. For eg, here it's PISCINE.

The only case you do not do it as it should be is when there is 2c components with same name.